Android.md

```diff
@@ -21,4 +21,11 @@
 1.  [Google CodeLabs](https://codelabs.developers.google.com/?cat=Android)
 
 ## For Experts
+### Newsletters (Experts must follow to remain updated with the latest trends)
```

Comment:
We are limiting this repo to resources and not newsletters, Let's keep it that way?